### DISCUSSION TITLE
Cannot retrieve the latest commit at this time - Failed to load latest commit information.

### DISCUSSION THREAD

**Discussion by sdancer75** [community/community#23320] [New to GitHub]
<p>Hi,</p>
<p>In a new repository, I get the error </p>
<p><span>Cannot retrieve the latest commit at this time.</span></p>
<p><span>Failed to load latest commit information.</span></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/4daeeafae67c67f2ef5c28e5d6358f2b9db69965.jpeg" data-download-href="/uploads/short-url/b5dwyoG2sE7GPxoIzlGgojBfSgR.jpg?dl=1" title="Screenshot_12.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10809692/181076130-1c445f47-6063-4967-996a-b146d6a24d28.jpg" alt="Screenshot_12.jpg" data-base62-sha1="b5dwyoG2sE7GPxoIzlGgojBfSgR" width="690" height="136" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/4daeeafae67c67f2ef5c28e5d6358f2b9db69965_2_690x136.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/4daeeafae67c67f2ef5c28e5d6358f2b9db69965_2_1035x204.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/1X/4daeeafae67c67f2ef5c28e5d6358f2b9db69965.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/4daeeafae67c67f2ef5c28e5d6358f2b9db69965_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_12.jpg</span><span class="informations">1276×252 40.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>What am I doing wrong? How can I fix this?</p>
<p>Regards,</p>

**Comment by logankilpatrick** [comment]
<p>Hey <a class="mention" href="/sdancer75">@sdancer75</a>, this is likely a temporary issue related to your browser or network connection. Can you try clearing your browser cache and then attempt to access the repo again? </p>

**Comment by sdancer75** [comment, marked as answer]
<p>Hi,</p>
<p>The root of the problem was the UBlock Origin Extension of the Chrome.</p>
<p><a href="https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=el" rel="nofollow noopener">https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=el</a></p>

**Comment by kixxpy** [comment]
I removed the extension in Google Chrome, a regular mini-game that caused everything to break
